src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
The 'body' tag is being styled directly, which is generally not recommended. It would be better to use a class to apply styles to the body. This is a code style issue.